Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -42,14 +42,11 @@ description: 'This playbook addresses the following alerts:
   Remediation:
 
 
-  - With analyst approval, disable the user in Active Directory if user-related anomalies
-  are found and the alert is a True Positive.
+  - With analyst approval, disable the user in Active Directory if user-related anomalies are found and the alert is a True Positive.
```

Comment:
With analyst approval, disable the user in Active Directory if user-related anomalies are found and the alert is a true positive.